### PR TITLE
squelch link script output

### PR DESCRIPTION
**Pull request by bollwyvl**
Nobody likes the output from the post-link scripts from `jupyter nbextension enable`. This redirects any such output.

I'll be doing these for the other extensions as well :)

### CODE REVIEW

**Comment by jakirkham** [comment]
This is under discussion. ( https://github.com/conda-forge/conda-forge.github.io/issues/173 )

**Comment by jakirkham** [comment]
Oops, missed you had this PR open still. Do you want to absorb my changes from PR ( https://github.com/conda-forge/pythreejs-feedstock/pull/4 )?

**Comment by SylvainCorlay** [comment]
@jakirkham I like the echo off thing better. I guess we can close this.

**Comment by jakirkham** [comment]
Oh, will this make the redirection unnecessary? I thought it only got rid of the commands from showing up. Please let me know if I'm misunderstanding, which may very well be the case.

**Comment by SylvainCorlay** [comment]
I actually don't know. I thought it served purpose. I guess not...

**Comment by jakirkham** [comment]
Sorry by "it", I meant `@echo off`.

**Comment by jakirkham** [comment]
According to this [doc](https://technet.microsoft.com/en-us/library/bb490897.aspx). It seems like you need both as their example still prints to the screen.

**Comment by jakirkham** [comment]
Merged into PR ( https://github.com/conda-forge/pythreejs-feedstock/pull/4 ).

File: recipe/post-link.bat [hunk 1]
```diff
@@ -1 +1 @@
-"%PREFIX%\Scripts\jupyter-nbextension.exe" enable pythreejs --py --sys-prefix && if errorlevel 1 exit 1
+"%PREFIX%\Scripts\jupyter-nbextension.exe" enable pythreejs --py --sys-prefix  > /dev/null 2>&1 && if errorlevel 1 exit 1
```
Comment (jakirkham) [inline recipe/post-link.bat:1]: As FYI, `> /dev/null 2>&1` should be `> NUL 2>&1`.

**Pull request closed without merging**